### PR TITLE
Missing reference docs for kernel.finish_request event

### PR DESCRIPTION
**Pull request by acrobat**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes - missing docs (PR symfony/symfony#8904) |
| Applies to | Event was added in 2.4 so added to **2.7** docs |
| Fixed tickets | #6151 |

### CODE REVIEW

**Comment by JHGitty** [comment]
Thanks!

File: reference/events.rst [hunk 1]
```diff
@@ -158,6 +158,25 @@ Listener Class Name
 :class:`Symfony\\Component\\HttpKernel\\EventListener\\StreamedResponseListener`     -1024
 ===================================================================================  ========
 
+``kernel.finish_request``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Event Class**: :class:`Symfony\\Component\\HttpKernel\\Event\\FinishRequestEvent`
+
+The purpose of this event is to allow you to reset the global and environmental state of
+the application, when it was changed during the request.
```
Comment (WouterJ) [inline reference/events.rst]: I would change "when it was changed during the request" to "after a sub-request has finished."

Maybe add the example of the translator listener: Resetting the locale to the locale of the parent request when finishing a sub-request.

**Comment by acrobat** [comment]
@WouterJ fixed your comment and added the translator listener example

File: reference/events.rst [hunk 2]
```diff
@@ -158,6 +158,35 @@ Listener Class Name
 :class:`Symfony\\Component\\HttpKernel\\EventListener\\StreamedResponseListener`     -1024
 ===================================================================================  ========
 
+``kernel.finish_request``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Event Class**: :class:`Symfony\\Component\\HttpKernel\\Event\\FinishRequestEvent`
+
+The purpose of this event is to allow you to reset the global and environmental state of
+the application after a sub-request has finished::
```
Comment (WouterJ) [inline reference/events.rst]: I suggest to change this to "after a sub-request has finished. For example, the translator listener uses it to reset the locale to the locale of the parent request:". However, that can also be done by us after merging, so don't worry if you have no time to update the PR.

**Comment by WouterJ** [comment]
Cool. Good job! :+1: from me

**Comment by xabbuh** [comment]
Thank you @acrobat for this nice pull request. I have merged your changes and slightly tweaked the description in 4cf2d0b609d8a15546c48b9326e4b4d1b07d06fe.

**Pull request merged**